src/gui/styles/dark/DarkStyle.cpp

```diff
@@ -89,6 +91,18 @@ QPalette DarkStyle::standardPalette() const
     return palette;
 }
 
+QIcon DarkStyle::standardIcon(StandardPixmap sp, const QStyleOption* opt, const QWidget* widget) const
+{
+    switch (sp) {
+    case SP_ToolBarHorizontalExtensionButton:
+        return resources()->icon("chevron-double-down");
+    case SP_ToolBarVerticalExtensionButton:
```

Comment:
If this is so embarrassingly simple, shouldn't we be also doing this for other standard icons such as the clear field icon? Right now those are custom actions, because the standard icons are useless.

Reply:
Great point and yes